tools/net/simple_file_transfer.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python3
+############################################################################
+#
+# Copyright 2020 Samsung Electronics All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+############################################################################
+import socket
+import os
+import argparse
+import logging
+import sys
+
+LOG = logging.getLogger(__name__)
+
+PORT = 5555        # The port used by the server
+
+
+def send_file(target_addr, send_file_name, target_file_name):
+
+    with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
+        s.connect((target_addr, PORT))
+        direction = 1
+        s.sendall(direction.to_bytes(4, byteorder='big'))
+        s.sendall(len(target_file_name).to_bytes(4, byteorder='big'))
+        s.sendall(target_file_name.encode())
+        try:
+            file_size = os.path.getsize(send_file_name)
+            LOG.info("file size: %d", file_size)
+            s.sendall(file_size.to_bytes(4, byteorder='big'))
+        except os.error:
+            LOG.exception("file is not exist")
+
+        with open(send_file_name, 'r') as infile:
+            data = infile.read()
+            s.sendall(data.encode())
+            LOG.info('File transfering is done')
+
+
+def receive_file(target_addr, send_file_name, target_file_name):
+    with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
+        s.connect((target_addr, PORT))
+        direction = 0
+        s.sendall(direction.to_bytes(4, byteorder='big'))
+        s.sendall(len(target_file_name).to_bytes(4, byteorder='big'))
+        s.sendall(target_file_name.encode())
+
+        data = s.recv(4)
+        file_size = socket.ntohl(int.from_bytes(data, sys.byteorder))
+        LOG.info("file size: %d", file_size)
+        with open(send_file_name, 'wb') as infile:
+            data = s.recv(file_size)
+            infile.write(data)
+            LOG.info("File receiving is done")
+
+    return
+
+
```

Comment:
Is a double line space required?
If not could you remove one please?

Reply:
it's recommended to set double line space between functions.